src/Source/AbstractPropertySourceIterator.php

```diff
@@ -33,7 +33,7 @@ abstract class AbstractPropertySourceIterator implements SourceIteratorInterface
     ];
 
     /**
-     * @var IterableResult
+     * @var IterableResult|null
```

Comment:
How does this relate to `rewind()` / this commit?

Reply:
Why is that wrong? Looking at `DoctrineDBALConnectionSourceIterator::__construct()`, it seems to me that this property can be null.

Reply:
Yes, it can be null. That's why I allow the null value.

Reply:
Oh sorry I read this the wrong way!